share/scripts/inverse/run_gromacs.sh

```diff
@@ -28,6 +28,27 @@ EOF
    exit 0
 fi
 
+tasks=$(get_number_tasks)
+
+mdrun_opts="$(csg_get_property --allow-empty cg.inverse.gromacs.mdrun.opts)"
+
+if [[ ${mdrun_opts} == *multidir* ]]; then
+
+  mdirs=$(echo ${mdrun_opts} | grep -o "sim" | wc -l)
```

Comment:
Honestly, I would prefer an user option, instead of relying on a hard-coded pattern (`sim`) in the options.

I would do that in the same spirit as the `pre` simulation feature has its own top, gro, mdp files, but fails back to the main one if not specified.